frontend/lib/Models/members_list_provider.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:adpv_frontend/DataModels/group_data.dart';
+import 'package:adpv_frontend/DataModels/member_summary.dart';
+import 'package:flutter/material.dart';
+
+enum SortingField { id, mail }
+
+enum SortingOrder { asc, desc }
+
+class Sorting {
+  SortingField field = SortingField.id;
+  SortingOrder order = SortingOrder.asc;
```

Comment:
Czy w jakiś wizualny sposób mamy zaznaczone na ekranie, jaki porządek sortowania mamy aktualnie wybrany (np. jakaś strzałka w górę/dół)?

Reply:
mogę sprobować ale nie obiecuję, bo jak trzeba bd setState użyć to sie to wszystko wyjebie